java/flight/flight-core/src/main/java/org/apache/arrow/flight/grpc/AddWritableBuffer.java

```diff
@@ -70,8 +70,8 @@ public class AddWritableBuffer {
       tmpBufChainOut = tmpBufChainOut2;
 
     } catch (Exception ex) {
-      new RuntimeException("Failed to initialize AddWritableBuffer, falling back to slow path", ex)
-          .printStackTrace();
+      throw new RuntimeException(
+          "Failed to initialize AddWritableBuffer, falling back to slow path", ex);
```

Comment:
I think this intentionally did not throw, as evidenced by the "falling back" message